### DISCUSSION TITLE
Anyone willing to help me starting development on this project?

### DISCUSSION THREAD

**Discussion by ignazioc** [jonaswinkler/paperless-ng#516] [Support]
I started using this software and I love it. I'd like to contribute but I'm not expert with python projects of this size. Anyone willing to do some onboarding? :P

**Comment by shamoon** [comment]
Not sure exactly what you’re asking, how to get a dev environment setup? 

There are lots of different ways to contribute, did you see https://github.com/jonaswinkler/paperless-ng#feel-like-helping-out (and the documentation ie https://paperless-ng.readthedocs.io/en/latest/extending.html )?

**Reply by ignazioc** [reply]
> how to get a dev environment setup?

yep exactly. prepare the dev enviroment, running tests, start the UI...

**Reply by shamoon** [reply]
Yea I kinda figured it out from documention but fumbled around a bit. PyCharm and Anaconda were helpful for me

**Comment by jonaswinkler** [comment]
This isn't exactly a large project though :) At it's core, it's fairly simple.

I'll get the development documentation updated. Configuring the development environment is pretty tough given the sparse instructions right now.

I'll also try to get the core components documented.

**Reply by ignazioc** [reply]
thanks @jonaswinkler !
